example/general_L10n_SG.md

```diff
@@ -0,0 +1,131 @@
+
+If the label is File, and the accesskey is `F`, it will be displayed as `File` on Windows and Linux, with an underscored `F`. If the accesskey was `O`, so a character not available in the original label, it will be displayed underlined between parenthesis: `File (O)`.
+
+One important thing to determine is if, for your locale, it makes sense to have localized accesskeys: for example, if most users will use a keyboard with a different layout (English), it might make sense to keep the English original accesskey instead of using a letter available in your localization.
+
+Accesskeys, like commandkeys, have their own lines within .dtd and .properties files and are usually identified by .accesskey in the string ID.
+
+### Variables
+
+Variables should never be translated. You can recognize a variable within a string by its beginning with a specific character (e.g., `$`, `#`, `%`, etc.) followed by a combination of words without spacing. For example, `$BrandShortName` and `%S` are variables.  You can move a variable around within a string, if the translation of the string requires it.
+
+#### Cross-references and links
+
+Cross-references (or links) must be consistently used within a translation. If a text contains a hyperlink URL to a support article in English, the translation should also contain a hyperlink to a translation of that support article (if available) or the English version. Links should not redirect to other pages nor should they be broken and unusable.
+
+### Brands, copyright, and trademark
+
+Brand names, as well as copyright and trademarks should never be translated, nor transliterated into a non-Latin based script. See the [branding guidelines](general_L10n_SG.md#branding) for more details.
+
+## Emphasis guidelines
+
+### Title Case
+
+Title case is a capitalization style that includes the following rules:
+
+* Always capitalize the first and last word in the text element, regardless of part of speech or other rules below
+* For words in between the first and last, capitalize all nouns, verbs, adverbs and pronouns
+* Capitalize conjunctions of four or more letters, but also capitalize if
+* Capitalize prepositions for four or more letters
+* Do not capitalize articles, prepositions of three or fewer letters, and conjunctions of three or fewer letters (but not if)
+* A note about hyphens: Capitalize the second word in a hyphenated compound if it would normally be capitalized as a single word according to title case rules. So, capitalize the S in Quick-Search Options, lowercase the o in Add-on Choices.
+
+Example: `All Your Activity`
+
+### Sentence Case
+
+Sentence case is a capitalization style where the first word of the text element is capitalized as well as any words that are normally capitalized.
+
+Example: `Sign in to continue to Firefox Sync`
+
+### All Caps
+
+All Caps is a style where every letter is capitalized. Use these for all buttons that contain text.
```

Comment:
The last sentence is not really true.

Reply:
Good catch. Last sentence is taken from Photon Copy Rules and was only supposed to appear in English example